README.md

```diff
@@ -1,7 +1,51 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Expresso Website
 
-## Getting Started
+"Expresso: The Art and Literary Club of NIT Patna - A vibrant website fostering creativity, literature, and artistry within the NIT Patna community."
 
+
+Expresso is the club where literature and art collide, creating a vibrant community for passionate students. It's a dynamic hub designed to nurture and showcase your talents, whether in crafting words or painting visuals.  Connect with a supportive community of fellow students, writers, and artists, leaving an indelible mark in this enchanting space. From diverse opportunities to skill development workshops, Espresso Club is more than a club – it's a journey of creative discovery. Contribute to the college's cultural tapestry, organize events, participate in campus initiatives, and make a lasting impression. 
+
+
+## Table of content
+
+- Home
+- Events
+- Team
+- Art Collection
+- Literature
+- Alumni
+
+## Installation Instructions
+
+
+Fork the repository-
+
+```bash
+  1.Click the "Fork" button
+  2.You'll see a copy of the original repository in your account, with your username as the owner. 
+  You can now make changes to this forked repository without affecting the original repository.
+
+```
+
+Clone the project
+
+```bash
+  git clone https://github.com/expressonitp/expresso_website.git
+```
+
+Install dependencies
+
+```bash
```

Comment:
remove npm install next@latest line and also the server line as there is no server